apps/files/src/components/FileEntry/FileEntryPreview.vue

```diff
@@ -50,6 +50,10 @@
 			:aria-label="t('files', 'Favorite')">
 			<FavoriteIcon v-once />
 		</span>
+
+		<OverlayIcon :is="fileOverlay"
+			v-if="fileOverlay"
+			class="files-list__row-icon-overlay" />
```

Comment:
Alternatively:
```suggestion
		<PlayCircleIcon v-if="isLivePhoto" class="files-list__row-icon-overlay" />
```

Then drop `fileOverlay` and add a computed:
```js
// computed: {
isLivePhoto() {
    return isLivePhoto(this.source)
}
```

Reply:
No, as we want a generic solution, like for folders